armedilla_alliyah_samantha/index.html

```diff
@@ -64,36 +64,46 @@ <h2>My thoughts about F2F class</h2>
     </div>
 
     <div class="segment">
-      <h2>Comments</h2>
-      <div class="comments-container">
-        <div class="comments-list">
-          <div class="sort-options">
-            <label for="sort_order">Arrange By:</label>
-            <select id="sort_order">
-              <option value="asc">Oldest</option>
-              <option value="desc">Newest</option>
-            </select>
-          </div>
-          <div id="comment_list"></div>
-        </div>
-        <div class="form-container">
-          <h3>Add Comment</h3>
-          <form id="comment_form" class="form">
-            <div class="form-group">
-              <label for="name_input">Name:</label>
-              <input type="text" id="name_input" name="name" required />
-            </div>
-            <div class="form-group">
-              <label for="comment_input">Comment:</label>
-              <textarea id="comment_input" name="comment" required> </textarea>
-            </div>
-            <div class="form-group">
-              <button type="submit" id="submit_btn">Submit Comment</button>
+      <h2>Comment Section</h2>
+      <p>Your goals reflect a clear sense of purpose, and 
+        I'm here to support you. - Rosario
+      <p>For now its a goal, someday it will be a memory - Gasta</p>
+      <p>I hope you achieve this all someday, Alliyah - Kian</p>
+      <p>Live long and prosper! - Jai</p>
+      <p>Lets go!! - John Deniel</p>
+    </div>
+
+    <div class="segment">
+        <h2>Comments</h2>
+        <div class="comments-container">
+          <div class="comments-list">
+            <div class="sort-options">
+              <label for="sort_order">Arrange By:</label>
+              <select id="sort_order">
+                <option value="asc">Oldest</option>
+                <option value="desc">Newest</option>
+              </select>
             </div>
-          </form>
+            <div id="comment-list"></div>
```

Comment:
Hi, good morning! Would you mind making new changes in naming ids? All IDs should be in snake_case, for example:

❌ id="comment-list"
✅ id="comment_list"

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

Reply:
Hello, you haven't changed the naming ID here yet.